app/actors/hyrax/actors/file_actor.rb

```diff
@@ -100,29 +100,28 @@ def normalize_relation(relation)
         end
 
         def normalize_relation_for_active_fedora(relation)
-          return relation if relation.is_a? Symbol
           return relation.to_sym if relation.respond_to? :to_sym
 
-          # TODO: whereever these are set, they should use FileSet.*_use... making the casecmp unnecessary
-          return :original_file if relation.to_s.casecmp(Hyrax::FileSet::ORIGINAL_FILE_USE.to_s)
-          return :extracted_file if relation.to_s.casecmp(Hyrax::FileSet::EXTRACTED_TEXT_USE.to_s)
-          return :thumbnail_file if relation.to_s.casecmp(Hyrax::FileSet::THUMBNAIL_USE.to_s)
-          :original_file
+          case relation
+          when Hyrax::FileMetadata::Use::ORIGINAL_FILE
+            :original_file
+          when Hyrax::FileMetadata::Use::EXTRACTED_TEXT
+            :extracted_file
+          when Hyrax::FileMetadata::Use::THUMBNAIL
+            :thumbnail_file
```

Comment:
As I was exploring inconsistencies documented in Issue #4228, I noticed that these are using symbols that I think are incorrect.  I'm pretty sure these should be :extracted_text and :thumbnail.

This part of why I propose putting any conversions between symbol and RDF::URI in a single place (e.g. FileMetadata::Use module).

Reply:
i also noticed that the symbols were inconsistent across modules. i'm not sure that means they are incorrect(?). these are the symbols that pre-date this PR, right?

unless they got mixed up somewhere else recently, they should be the ones callers are already passing in.

in any case, i do think we should hold changes to behavior around these symbols for resolution as part of #4228. for now, the behavior exists in two places because it behaves slightly differently across them, and the goal here is to preserve that faithfully.

Reply:
It looks like these were incorrectly entered in PR #2746 which was one of the first PRs for valkyrizing files.  I don't think they have surfaced as a problem because Hyrax generally already passes relation as a Symbol unless it is a custom relation, so the conversion probably has never been used in a Hyrax app.

Reply:
Looking back on PR #2746 for this part handling the conversion from URI to Symbol and vice versa, I am somewhat uncomfortable with the default being original file.  I'm wondering what happens if a custom file type is passed in for relation.

I'm can update #4228 with this question or create a separate issue.  It seems a bit beyond the refactoring for consistency issue.

Reply:
re: `:original_file` i think this is just a straight refactor, and i'd prefer not to let its scope creep.

i'm in favor of wrapping any remaining issues into #4228.

Reply:
Done... #4228 is updated.